openedx/adg/lms/applications/constants.py

```diff
@@ -71,6 +72,11 @@
 
 ADG_ADMIN_GROUP_NAME = 'ADG Admins'
 
+MAX_NUMBER_OF_REFERENCES = 2
+MAX_REFERENCE_ERROR_MSG = format_lazy(
+    f'References against a user application cannot be more than {MAX_NUMBER_OF_REFERENCES}.'
+)
```

Comment:
Generally, we write error message of this kind in the form, 

> This field cannot exceed...

Perhaps we can phrase this message in that way as well for consistency.

Reply:
Actually this error message is not related to the field. It is a non-field error message that is displayed when a client attempts to add more than 2 references against a user application.

Reply:
A: Making it translatable doesn't cost us anything, so I think It's better if we do. I understand that the error message would be displayed if someone is calling it directly via API, and it is not exactly user facing, since we do not allow the user from the front end to add more than 2 references. The translation could be of potential help for the future, but I can remove it if you want me to.

Reply:
If we would go with a validator, let's move the error inside their too because not something we could reuse at multiple places and our non-field error message are also in the validators. Also is there any specific reason behind lazy formatting it? like there is a constant we are using in there and it would be ready even at application load time.

Reply:
@ahmed-faraz46 I am now using a validator, but I'm still keeping the error message in the `constants.py` file since I'm also using it in the unit test. I've removed the lazy formatting, thanks for pointing it out.